django/text_extractor/tasks.py

```diff
@@ -0,0 +1,35 @@
+import os
+from io import BytesIO
+
+from django.contrib.auth import get_user_model
+from django.core.files.base import ContentFile
+from django.core.files.uploadedfile import InMemoryUploadedFile
+
+from celery import shared_task
+
+from otto.secure_models import AccessKey
+from otto.utils.common import display_cad_cost, file_size_to_string
+from text_extractor.models import OutputFile, UserRequest
+
+from .utils import create_searchable_pdf, shorten_input_name
+
+
+# passing the OCR method to celery
+@shared_task
+def process_ocr_document(file_content, file_name, merged, idx):
+
+    file = InMemoryUploadedFile(
+        file=BytesIO(file_content),
+        field_name=None,
+        name=file_name,
+        content_type="application/pdf",
+        size=len(file_content),
+        charset=None,
+    )
```

Comment:
celery task created: 
-Temporary file space created within celery (easier to find rather than passing the file path)
-main OCR method given to celery - "create_searchable_pdf"
-everything else remains same